src/Object/Post.php

```diff
@@ -667,50 +686,78 @@ private function getEmojis(array $item): array
 				case Activity::ANNOUNCE:
 					$title = DI::l10n()->t('Reshared by: %s', $actors);
 					$icon  = ['fa' => 'fa-retweet', 'icon' => 'icon-retweet'];
+					$verb  = Activity::ANNOUNCE;
```

Comment:
This can be simplified by using `$element['verb']` directly. Either initialize `$verb` with the value of `$element['verb']` or directly set `$element['verb']` with the value of `$element['emoji']` in the default case before using it in line 740.

Reply:
I simplified the code.

Reply:
Did you see I replied in the other thread?

Reply:
Which other thread?